NetSDK/Services/Parsing/Classes/SplitParser.cs

```diff
@@ -17,7 +17,9 @@ public ParsedSplit Parse(Split split)
         {
             try
             {
-                if (split.status != StatusEnum.ACTIVE)
+                StatusEnum result;
+                var isValidStatus = Enum.TryParse(split.status, out result);
+                if (!isValidStatus || result != StatusEnum.ACTIVE)
```

Comment:
I status is not ACTIVE or status is not a valid status (known and defined in enum) it will behave as if status is not ACTIVE.
It prevents the sdk to work with a split with a status different than ACTIVE.